core-bundle/contao/templates/elements/ce_toplink.html5

```diff
@@ -1,6 +1,16 @@
+<?php
+
+$this->wrapperAttributes = $this
+    ->attr($this->cssID)
+    ->addClass([$this->class, 'block'])
+    ->addStyle($this->style)
+    ->mergeWith($this->wrapperAttributes)
+;
+
+?>
 
 <!-- indexer::stop -->
```

Comment:
@fritzmg Why did you not extend from `block_unsearchable` here?

Reply:
Because this template does not use the `headline` and `content` block.